source/components/messageLog/messageLog.service.ts

```diff
@@ -1,6 +1,11 @@
+'use strict';
```

Comment:
FYI - we're no longer using `'use strict'` at the top of our files. This is a detail that typescript handles for us now, so we shouldn't need to worry about it anymore